specs/core/0_beacon-chain.md

```diff
@@ -587,11 +587,11 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 ```python
 {
     # Previous fork version
-    'pre_fork_version': 'uint64',
+    'previous_version': 'uint64',
     # Post fork version
-    'post_fork_version': 'uint64',
+    'post_version': 'uint64',
```

Comment:
`current_version` to align with `previous_version`?

Reply:
agreed